editor/styling/stylesheets/modules/_properties.scss

```diff
@@ -1,7 +1,4 @@
 #properties {
-    HBox {
-        -fx-spacing: 0;
-    }
```

Comment:
I was going crazy because of this at some point. I wanted spacing, but there was none, even though I had CSS style classes that only added spacing. But yeah, you can target Java classes with CSS...